### PR TITLE
[FORMS-2970] On next button click added validation of that wizard panel

### PR DESCRIPTION
**Pull request by deepprakash345**
[FORMS-2970] On next button click added validation of that wizard panel

Added test cases for following scenarios:-

- validation error occurs for all the member of that particular wizard panel . Other wizard panels are not validated at the 
same step.
- next button allows to move to next tab when all validations are passed.
- previous button allows to move to previous tab even if validation failure is there.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js [hunk 1]
```diff
@@ -276,12 +276,18 @@
...
-                    var totalTabs = tabs.length;
-                    if (activeIndex < totalTabs - 1) {
-                        this.#navigateAndFocusTab(activeIndex + 1);
+            var activeTabElement = this.#getCachedTabs()[activeIndex];
+            var activeChildId = activeTabElement.id.substring(0, activeTabElement.id.lastIndexOf(Wizard.#tabIdSuffix));
+            var activeChildView = this.getChild(activeChildId);
+            var validationErrorList = activeChildView.getModel().validate();
+            if (validationErrorList === undefined || validationErrorList.length == 0) {
+                var tabs = this.#getCachedTabs();
+                if (tabs) {
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:285]: if (tabs && Array.isArray(tabs))

Comment (deepprakash345) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:285]: removed the unnecessary isArray checks

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js [hunk 2]
```diff
@@ -276,9 +276,13 @@
...
-            var tabs = this.#getCachedTabs();
-            if (tabs) {
-                if (Array.isArray(tabs)) {
+            var activeTabElement = this.#getCachedTabs()[activeIndex];
+            var activeChildId = activeTabElement.id.substring(0, activeTabElement.id.lastIndexOf(Wizard.#tabIdSuffix));
+            var activeChildView = this.getChild(activeChildId);
+            var validationErrorList = activeChildView.getModel().validate();
+            if (validationErrorList === undefined || validationErrorList.length == 0) {
+                var tabs = this.#getCachedTabs();
+                if (tabs) {
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:285]: if (tabs && (activeIndex < tabs.length - 1))

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js [hunk 3]
```diff
@@ -291,11 +295,10 @@
             var activeIndex = this.#_active;
             var tabs = this.#getCachedTabs();
             if (tabs) {
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:297]: if (tabs && (activeIndex > 0))

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js (hunk 2 above)
Comment (sakshi-arora1) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:285]: merge if, avoid arrow code

Comment (deepprakash345) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js]: done

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js (hunk 3 above)
Comment (deepprakash345) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js]: done

**Review by sakshi-arora1 — APPROVED** [review]

**Pull request merged**